src/map/utils/battleutils.cpp

```diff
@@ -1808,7 +1808,7 @@ namespace battleutils
     *																		*
     ************************************************************************/
 
-    int32 TakePhysicalDamage(CBattleEntity* PAttacker, CBattleEntity* PDefender, int32 damage, bool isBlocked, uint8 slot, uint16 tpMultiplier, CBattleEntity* taChar, bool giveTPtoVictim, bool giveTPtoAttacker, bool isCounter)
+    int32 TakePhysicalDamage(CBattleEntity* PAttacker, CBattleEntity* PDefender, uint16 attackType, int32 damage, bool isBlocked, uint8 slot, uint16 tpMultiplier, CBattleEntity* taChar, bool giveTPtoVictim, bool giveTPtoAttacker, bool isCounter)
```

Comment:
I actually had it that way initially and it yelled, a lot. One of the errors was "cannot convert uint16 to PHYSICAL_ATTACK_TYPE so I changed it to a uint16 and it worked fine afterwords. I'll try to get it to work as a PHYSICAL_ATTACK_TYPE again.